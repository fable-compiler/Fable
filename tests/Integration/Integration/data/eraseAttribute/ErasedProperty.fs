module ErasedProperty

open Fable.Core


// This test is to make sure that we are retrieving the Erase attribute from the erased property
// and not the declaring type

type internal LanguageInjectionAttribute() =

    [<Erase>]
    member val Prefix = "" with get, set
