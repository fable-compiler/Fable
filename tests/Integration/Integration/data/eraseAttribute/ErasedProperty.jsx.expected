import { class_type } from "./fable_modules/fable-library-js.5.0.0-alpha.3/Reflection.js";

export class LanguageInjectionAttribute {
    constructor() {
        this["Prefix@"] = "";
    }
}

export function LanguageInjectionAttribute_$reflection() {
    return class_type("ErasedProperty.LanguageInjectionAttribute", undefined, LanguageInjectionAttribute);
}

export function LanguageInjectionAttribute_$ctor() {
    return new LanguageInjectionAttribute();
}

